src/EnergyPlus/CMakeLists.txt

```diff
@@ -570,11 +570,16 @@ install( TARGETS energyplusapi
   LIBRARY DESTINATION ./ 
 )
 
-if (CMAKE_COMPILER_IS_GNUCXX AND ENABLE_LTO)
+if (CMAKE_COMPILER_IS_GNUCXX)
```

Comment:
@nealkruis That was also my expectation. Unless @lefticus or @kbenne (sorry to ping you guys again) weigh in with some theory as to what's going on here, I'm inclined to close the pull request. I definitely want the MinGW support, so I think we should keep the branch and revisit it later.